src/IDPDiscoveryController.js

```diff
@@ -103,6 +103,10 @@ export default PrimaryAuthController.extend({
       );
 
       // Events to set the transaction attributes on the app state.
+      this.listenTo(primaryAuthModel, 'error', function(src, errObj) {
+        this.toggleButtonState(false);
+        this.model.trigger('error', this.model, errObj);
+      });
       this.addModelListeners(primaryAuthModel);
```

Comment:
Looks like the primary reason was we forgot to propagate certain events from `primaryAuthModel` to `this.model` which has listened to many events already.
Not for this PR but I wonder if we'd like to try a follow up refactoring that simply bubble up events (errors, setTransaction etc) would be good enough without even invoke `addModelListeners`.

Reply:
I was thinking the same, either bubble up events somehow or override `this.model` instead of creating a new model here. I didn't want to do it now since this is urgent and we would need more testing if we refactor more. I created https://oktainc.atlassian.net/browse/OKTA-463593 to follow up, take a look @haishengwu-okta and add any more details if I missed something